### DISCUSSION TITLE
Import not working after minimatch upgrade to ^6.1.6

### DISCUSSION THREAD

**Discussion by RahulGautamSingh** [editorconfig/editorconfig-core-js#114] [Q&A] [answered]
Hey guys,

We are using editorconfig in Renovate as dep and we use minimatch as well. In a recent PR we tried upgrading minimatch to `6.1.6` but this caused import issues with editorconfig as it uses minimatch as well but `5.1.0` version. In recent upgrades minimatch made some changes to the way it is exported/imported, so maybe a small refactor needs to be made to the code.

I tried changing `import minimatch from "minimatch"` to `import minimatch = require("minimatch")` locally and it seemed to work.
Coould you please look into it.

**Comment by hildjj** [comment, marked as answer]
Working on a fix here: https://github.com/editorconfig/editorconfig-core-js/pull/115

**Comment by hildjj** [comment]
Fixed in release 1.0.2
